### PR TITLE
Lock SQLite3 to < 1.4

### PR DESCRIPTION
**Pull request by mamhoff**
On my system, I couldn't get `rake db:create` to run on `master`, as the
version of SQLite3 in the Gemfile.lock would result in the following
error. It's likely this didn't cause issues before as CI runs on MySQL and the upgrade to 1.4.1 happened only 2 weeks ago.

https://stackoverflow.com/questions/54527277/cant-activate-sqlite3-1-3-6-already-activated-sqlite3-1-4-0

(Ok, not my question, it was 1.4.1 on my machine, but same thing).

Locking SQLite3 to '~> 1.3.6` did fix the issue. It'll probably a great
candidate to upgrade once we're on Rails 5.

### CODE REVIEW

File: Gemfile.lock [hunk 1]
```diff
@@ -583,4 +583,4 @@ DEPENDENCIES
   whenever
 
 BUNDLED WITH
-   1.16.1
+   1.17.1
```
Comment (paroga) [inline Gemfile.lock]: please keep this unrelated change out

**Comment by paroga** [comment]
committed as 5d82f16c70bd9077662f667e4fc0dd0a4b04915b

**Comment by mamhoff** [comment]
Thank you!
